openapi/components/schemas/Website.yaml

```
@@ -85,6 +85,10 @@ properties:
                 nullable: true
                 items:
                   type: string
+              fullPageRedirect:
+                type: boolean
+                description: Specifies whether the hosted payment form uses a full page redirect, or the default iframe modal, for approval URL redirects.
+                nullable: true
```

I wonder if it's better to specify the default value, for clearer documentation?  It's not needed as it's `false` by default.

```suggestion
                nullable: true
                default: false
```